.gitignore

```diff
@@ -165,3 +165,6 @@ dask-worker-space/
 # Sphinx docs & build artifacts
 docs/cudf/source/api_docs/generated/*
 docs/cudf/source/api_docs/api/*
+docs/cudf/source/user_guide/example_files/*
```

Comment:
I'd like to modify the `10min.ipynb` to put this in `example_outputs` so that all files are in the same output directory.

Reply:
Updated.